pkg/provider/data_source_cluster.go

```diff
@@ -165,6 +165,11 @@ func (c *ClusterData) Schema() *schema.Resource {
 				Type:        schema.TypeBool,
 				Computed:    true,
 			},
+			"project_id": {
+				Description: "BigAnimal Project ID.",
```

Comment:
is it worth validating the input here and make sure that it's in the form /prj_.*/ ?

Reply:
Aaand, we have our first `validators.go` as well as `validators_test.go` 😄